app/github/deployment_status_event.rb

```diff
@@ -16,6 +16,8 @@ def organization
   end
 
   def user
+    # TODO (rballestrini): can this be refactored to use the following?
+    # @user ||= User.find_by_github(event['deployment']['creator']['id'])
```

Comment:
Good catch. `find_by_github` and `find_by_github_account_id` are duplicates. We can make this change and remove `find_by_github_account_id`.

Reply:
I suspected as much, thank you for clarifying!